app/controllers/orga/seasons_controller.rb

```diff
@@ -48,16 +48,16 @@ def destroy
   # by opening and closing the corresponding links in the nav bar
   def switch_phase
     return if Rails.env.production?
-    @season = current_season
+    @switch = current_season.phase_switcher
```

Comment:
You can instantiate the class directly without calling a method on `current_season`. Now that there is a dedicated class for it, it doesn't have to be the `Season`'s concern to switch phases anymore.